lib/commands/mobile.js

```diff
@@ -0,0 +1,34 @@
+import { youiEngineKeycode, youiEngineKeycodear } from '../utils';
+import log from '../logger';
+
+
+let extensions = {};
+
+extensions.mobilePressButton = async function mobilePressButton (opts = {}) {
+  const {name} = opts;
+  if (!name) {
+    log.errorAndThrow('Button Name is mandatory');
+  }
+  let code = youiEngineKeycode.indexOf(name);
```

Comment:
Could you add a check for if the code is not found in the array? Ex. Invalid keycode [value] supplied.